modules/freewheelSSPBidAdapter.js

```diff
@@ -223,6 +223,20 @@ export const spec = {
       componentId: getComponentId(currentBidRequest.params.format)
     };
 
+    if (typeof currentBidRequest.params.gdpr !== 'undefined') {
```

Comment:
this wasn't done properly.  `buildRequests(bidRequests, bidderRequest)` has a second `bidderRequest` argument where you'll find the `gdpr` params.  please see the [doc](http://prebid.org/dev-docs/modules/consentManagement.html#buildrequests-integration).

Reply:
also, more specifically, I believe `gdpr_consent` could exist while `gdpr` is undefined.  So checking `gdpr !== 'undefined'` is not a good way to do this.

Reply:
Thx, I'll have a look.

Reply:
Changes have been made, thx again.